shell/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM mcr.microsoft.com/azurelinux/base/core:3.0
```

Comment:
should this be pinned?

Reply:
do you want me to pin this for reproducibility? or leave it unpinned so we always get the latest?

Reply:
let's pin this and then add a comment on top of this line to clarify which version of the image we are pulling

Reply:
thanks, updated!